File: src/main/kotlin/xingkong/loopicserver/picproc/RenameLinear.java
package xingkong.loopicserver.picproc;


import xingkong.loopicserver.picproc.multiDir.SingleDirRename;

import java.util.Scanner;

public class RenameLinear {
    public RenameLinear() {

    }

    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);
        SingleDirRename r = new SingleDirRename();
        while (true) {
            r.img_index = 0;
            r.file_num = 0;
            System.out.println("------------------------------------");
            in.useDelimiter("\n");
            String m = in.next();
            String pre = r.getNameRoot(6);
//            String pre = "fengjing";
//            int offset = in.nextInt();
            int offset = 0;
            System.out.println("Begin!");
            r.getFileListByDir(m);
            r.renameLinear(pre, offset);
            System.out.println("Success!!");
        }
    }
}

File: src/main/kotlin/xingkong/loopicserver/picproc/RenameLinear.kt
package xingkong.loopicserver.picproc


import xingkong.loopicserver.picproc.multiDir.SingleDirRename
import java.util.*

class RenameLinear {
    companion object {

        @JvmStatic
        fun main(args: Array<String>) {

            val `in` = Scanner(System.`in`)
            val r = SingleDirRename()
            while (true) {
                r.img_index = 0
                r.file_num = 0
                println("------------------------------------")
                `in`.useDelimiter("\n")
                val m = `in`.next()
                val pre = SingleDirRename.getNameRoot(6)
                //            String pre = "fengjing";
                //            int offset = in.nextInt();
                val offset = 0
                println("Begin!")
                r.getFileListByDir(m)
                r.renameLinear(pre, offset)
                println("Success!!")
            }
        }
    }
}
